### PR TITLE
Updates needed for the new sub version of experimental python sdk

### PR DESCRIPTION
**Pull request by lynochka**
Updates related to the new version of experimental sdk

### CODE REVIEW

File: contextualize_pnid_files.ipynb [hunk 1]
```diff
@@ -95,20 +95,20 @@
     "# overwrite the svg file if already uploaded\n",
     "svg_overwrite = True\n",
     "\n",
-    "for file in files:\n",
+    "for file in files[1:]:\n",
```
Comment (andeplane) [inline contextualize_pnid_files.ipynb]: Why skip [0]?

File: contextualize_pnid_files.ipynb [hunk 2]
```diff
@@ -63,7 +63,7 @@
    "source": [
     "# Get files with source\n",
     "source = \"some_source\"\n",
-    "files = client.files.list(source=source, limit=-1)"
+    "files = client.files.list(mime_type=\"application/pdf\", source=source, limit=-1)"
```
Comment (andeplane) [inline contextualize_pnid_files.ipynb]: This change is not reflected by the PR title. The PR title claims that you only fix getpass, so this belongs in another PR.

**Review by andeplane — CHANGES_REQUESTED** [review]
Nice! But the PR title claims this to only be about the getpass fix. Those other changes should be in a different PR so tracking changes is easier to do.

File: contextualize_pnid_files.ipynb (hunk 1 above)
Comment (lynochka) [inline contextualize_pnid_files.ipynb]:
```suggestion
    "for file in files:\n",
```

**Review by andeplane — APPROVED** [review]
